examples/webgl_lines_fat.html

```diff
@@ -160,6 +196,10 @@
 				// renderer will set this eventually
 				matLine.resolution.set( window.innerWidth, window.innerHeight ); // resolution of the viewport
 
+				// perform raycast after the resolution has been updated for the camera
+				// view we care about
+				doRaycast();
```

Comment:
This may be a similar case to what was discussed in #16423 where sprites needed more contextual about the camera being used in order to properly be raycast against. In this case the renderer canvas that is being clicked should dictate the resolution that is used when raycasting against the lines which means the renderer or the resolution should be available as context for the cast. Otherwise we have this scenario where the last call made to `renderer.render` will change the outcome of the raycast which would be confusing.

One solution is to add a `raycaster.resolution` or `raycaster.renderer` field which could be required when raycasting against these lines much like `raycaster.camera` but I'd be happy to hear other thoughts!